### PR TITLE
Update hawk version

### LINKED ISSUE #2874: Hawk dependency causes security risk
I recently ran a Node Security scan on a project that uses Request and found a security issue related to the version of Hawk (and subsequently Hoek) that Request is using.

request@2.83.0 > hawk@6.0.2 > hoek@4.2.0

https://nodesecurity.io/advisories/566
https://hackerone.com/reports/310439

### PR DESCRIPTION
**Pull request by santiagogil**
## PR Checklist:
- [x] I have run `npm test` locally and all tests are passing.
- [ ] I have added/updated tests for any new behavior.
       
- [ ] If this is a significant change, an issue has already been created where the problem / solution was discussed: [N/A, or add link to issue here]
       

## PR Description

Update hawk version in order to avoid a known vulnerability in hoek.
request@2.83.0 > hawk@6.0.2 > hoek@4.2.0

https://nodesecurity.io/advisories/566
https://hackerone.com/reports/310439

Closes #2874

### CODE REVIEW

**Comment by hanjukim** [comment]
Please merge for stopping security alerts!

**Comment by benwiggins** [comment]
Re-install your packages and/or check your lockfile. 
hawk@6.0.2 should be pulling in hoek@4.2.**1** which is not vulnerable.

**Comment by compulim** [comment]
`hoek@<5.0.3` is vulnerable, published at [CVE-2018-3728](https://nvd.nist.gov/vuln/detail/CVE-2018-3728).

Starting from `hawk@7.0.0`, they [start using](https://github.com/hueniverse/hawk/blob/v7.0.0/package.json) `hoek@5.x.x`.

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "forever-agent": "~0.6.1",
     "form-data": "~2.3.1",
     "har-validator": "~5.0.3",
-    "hawk": "~6.0.2",
+    "hawk": "~7.0.0",
```
Comment (fahrradflucht) [inline package.json]: Did you check that this is sufficient and doesn't break anything? Hawk did a major version upgrade for a reason. The api is now not callback based anymore [0].

**Comment by simov** [comment]
See https://github.com/request/request/pull/2943

**Pull request closed without merging**